python/GafferOSL/FocalBlur.gfr

```diff
@@ -0,0 +1,889 @@
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["BoxIn7"]["out"], 'noduleLayout:section', 'right' )
+__children["FocalBlur"]["BoxIn7"]["__uiPosition"].setValue( imath.V2f( 787.27417, 14.2964497 ) )
+__children["FocalBlur"]["blurMultiplier"].setValue( 5.0 )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["blurMultiplier"], 'noduleLayout:section', 'left' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["blurMultiplier"], 'description', 'A global multiplier on the amount of blur, multiplied onto whatever blur results from the DOF and Camera parameters.' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["blurMultiplier"], 'noduleLayout:visible', True )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["blurMultiplier"], 'layout:section', 'Settings' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["blurMultiplier"], 'layout:index', 2 )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["blurMultiplier"], 'nodule:type', '' )
+__children["FocalBlur"]["computedFocalLength"].setInput( __children["FocalBlur"]["SceneCameraExpression"]["__out"]["p2"] )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["computedFocalLength"], 'noduleLayout:label', 'focalLength.value' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["computedFocalLength"], 'description', "The focal length is one of the primary attributes of a real lens ( driven by cameraScene ).\n\nIn conjunction with the aperture, this defines the camera's field of view." )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["computedFocalLength"], 'layout:section', 'Settings' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["computedFocalLength"], 'layout:index', 15 )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["computedFocalLength"], 'nodule:type', '' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["computedFocalLength"], 'layout:visibilityActivator', 'cameraScene' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["computedFocalLength"], 'label', 'Focal Length' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["horizontalAperture"], 'layout:section', 'Settings' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["horizontalAperture"], 'layout:index', 16 )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["horizontalAperture"], 'nodule:type', '' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["horizontalAperture"], 'layout:visibilityActivator', 'cameraCustom' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["horizontalAperture"], 'description', "The aperture is one of the primary attributes of a real lens ( driven by cameraScene ).\n\nIn conjunction with the focal length, this defines the camera's field of view. For the purpose of computing focal blur, we just take the horizontal aperture, which we expect to correspond to the horizontal resolution of the input image." )
```

Comment:
This description includes "( driven by cameraScene )" while the one for `computedHorizontalAperture` does not...

Reply:
Think it'd be worth describing that "aperture" here refers to the horizontal film-back or sensor width rather than a primary attribute of a real lens (where it may be confused with f-stop?).